README.md

```diff
@@ -107,6 +107,7 @@ The following examples show how to use Easegress for different scenarios.
 - [LoadBalancer](./doc/cookbook/load_balancer.md) - A number of strategy of load balancing 
 - [Distributed Tracing](./doc/cookbook/distributed_tracing.md) - How to do APM tracing  - Zipkin.
 - [Kubernetes Ingress Controller](./doc/cookbook/k8s_ingress_controller.md) - How to integrated with Kubernetes as ingress controller
+- [Pipeline](./doc/cookbook/pipeline.md) - How to orchestrate HTTP filters for requests/responses handling
 - [Performance](./doc/cookbook/performance.md) - Performance optimization - compression, caching etc.
```

Comment:
`performance` should be before `pipeline` ;-)

Reply:
Ok, fix it already, thx.